### DISCUSSION TITLE
Problem With DataTable Row Multiple Selection

### DISCUSSION THREAD

**Discussion by arikardnoir** [primefaces/community#1463] [PrimeVue]
When im traing to use the multiselect on Database, and i select a
single row, he visually select all of them, but debugging i found out
that he select only one, i dont know why, ill add some attachment based
on the problem:
![Screenshot from 2024-03-16 23-32-38](https://github.com/primefaces/community/assets/18501593/1747ac20-b008-4e7e-a3c0-4118b155ad75)
![Screenshot from 2024-03-16 23-32-07](https://github.com/primefaces/community/assets/18501593/f710af1b-2dde-46be-846a-6a7b90e5112f)
![Screenshot from 2024-03-16 23-31-04](https://github.com/primefaces/community/assets/18501593/c54ac248-dc7f-43f2-9722-0375d0230fdf)

**Comment by tugcekucukoglu** [comment]
Could you attach a reproducer link?
